frida_tools/tracer.py

```diff
@@ -32,6 +33,8 @@ def _add_options(self, parser):
             pb = TracerProfileBuilder()
             def process_builder_arg(option, opt_str, value, parser, method, **kwargs):
                 method(value)
+            parser.add_option("-q", "--quiet", help="do not format agents output",
+                    action='callback', callback=process_builder_arg, callback_args=(pb.quiet,))
```

Comment:
Better to use `action='store_true'` and drop the `callback` and `callback_args`.